src/parsing/effect_parsing.hpp

```diff
@@ -1393,7 +1393,17 @@ struct effect_body {
 					context.compiled_effect.push_back(trigger::payload(it->second).value);
 				} else {
 					err.accumulated_errors +=
-							"annex_to effect given an invalid tag (" + err.file_name + ", line " + std::to_string(line) + ")\n";
+						"annex_to effect given an invalid tag (" + err.file_name + ", line " + std::to_string(line) + ")\n";
+					return;
+				}
+			} else if(value == "null") {
+				if(context.from_slot == trigger::slot_contents::nation)
```

Comment:
why are you checking the from slot? You can check the primary slot to thanos snap either a province, state, or nation, but the from slot shouldn't matter